atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CellDeleter.java

```diff
@@ -88,7 +93,7 @@ public Void apply(CassandraClient client) throws Exception {
                             pred.setColumn_names(Arrays.asList(colName));
                             Deletion del = new Deletion();
                             del.setPredicate(pred);
-                            del.setTimestamp(ts + 1);
+                            del.setTimestamp(deleteTimestampGetter.apply(ts));
```

Comment:
As mentioned by others, move the `deleteTimestampGetter.apply` outside the for loops for efficiency.